src/org/jitsi/impl/neomedia/transform/DiscardTransformEngine.java

```diff
@@ -63,11 +63,16 @@ public RawPacket reverseTransform(RawPacket pkt)
                 = (pkt.getFlags() & Buffer.FLAG_DISCARD) == Buffer.FLAG_DISCARD;
 
             long ssrc = pkt.getSSRCAsLong();
-            ResumableStreamRewriter rewriter = ssrcToRewriter.get(ssrc);
-            if (rewriter == null)
+
+            ResumableStreamRewriter rewriter;
+            synchronized(ssrcToRewriter)
```

Comment:
Are you sure about this? It is accessed by both the RTP and RTCP transformers, which IIRC are used by different threads.